app/metrics.yaml

```
@@ -6229,7 +6229,12 @@ browser.search:
     type: labeled_counter
     description: |
       Records counts of SERP pages with adverts displayed.
-      The key format is `<provider-name>`.
+      The key format is 
+      `<provider-name>.in-content.[sap|sap-follow-on|organic].[code|none](.[channel])?`, where
+      `provider-name` is the name of the provider,
+      `sap|sap-follow-on|organic` is the search access point,
+      `code` is set when the url matches any of the provider's code prefixes,
+      `channel` is set to the url "channel" query parameter.
```

```suggestion
      The key format is
      `<provider-name>.in-content.[sap|sap-follow-on|organic].[code|none](.[channel])?`, where:

      * `provider-name` is the name of the provider,
      * `sap|sap-follow-on|organic` is the search access point,
      * `code` is set when the url matches any of the provider's code prefixes,
      * `channel` is set to the url "channel" query parameter.
```

By using this format [already used here](https://github.com/mozilla-mobile/fenix/blob/e79a16f284df297059586ea3e99d67624499c70e/app/metrics.yaml#L104-L110) properties will show up in a list making it [easier to read in the tools](https://dictionary.telemetry.mozilla.org/apps/fenix/metrics/events_performed_search).